yarn.lock

```diff
@@ -10468,10 +10469,6 @@ yargs@~3.27.0:
     window-size "^0.1.2"
     y18n "^3.2.0"
 
-yarn@^1.5.1:
```

Comment:
Is this a side effect of removing `react-jsonschema-form`?

Reply:
I honestly do not know why this was in here before.